app/routes/company/components/CompaniesPage.tsx

```diff
@@ -188,36 +188,39 @@ const CompaniesPage = () => {
             Vis mer
           </Button>
         )}
-        <div className={!expanded ? utilities.hiddenOnMobile : undefined}>
-          <p className={styles.infoText}>
-            Trykk deg inn på en bedrift for å se hva slags type bedrift det er,
-            les mer om hva de jobber med og se hvor de holder til. Bla deg
-            gjennom en oversikt over tidligere eller kommende arrangementer og
-            se hvem som har jobbannonser ute for øyeblikket. Hvis du vil lese
-            mer om bedriften så kan du navigere deg til nettsiden deres via
-            linken.
-          </p>
-
-          <p className={styles.infoText}>
-            Savner du en bedrift? Savner du noe informasjon om en bedrift? Ta
-            kontakt med Bedkom, vi tar gjerne imot innspill!
-          </p>
-          <Button
-            flat
-            className={styles.readMore}
-            onPress={() => setExpanded(false)}
-          >
-            Vis mindre
-          </Button>
-        </div>
+        {expanded && (
+          <div className={!expanded ? utilities.hiddenOnMobile : undefined}>
```

Comment:
```suggestion
          <div className={utilities.hiddenOnMobile}>
```
No need for ternary after `expanded && `

Reply:
nice catch!